README.md

```diff
@@ -16,93 +17,23 @@ The **VMware.vSphereDSC** module contains resources for:
 - **vCenter** and **Host** settings
 
 For more information about all available **Resources**, please visit the **wiki** page of the repository: [Wiki](https://github.com/vmware/dscr-for-vmware/wiki).
+For more information about the **VMware.vSphereDSC** module please read here: [VMware.vSphereDSC info](https://github.com/vmware/dscr-for-vmware/blob/master/VMware.vSphereDSC.md).
```

Comment:
Maybe rewrite it to:

For more information about the **VMware.vSphereDSC** module, please visit: [VMware.vSphereDSC](https://github.com/vmware/dscr-for-vmware/blob/master/VMware.vSphereDSC.md).

The same goes for the VMware.PSDesiredStateConfiguration module.